Gemfile

```diff
@@ -122,8 +123,8 @@ group :development, :test do
   gem "capybara", "~> 3.13"
   gem "derailed", "~> 0.1"
   gem "erb_lint", "~> 0.0", require: false
-  gem "faker", git: "https://github.com/stympy/faker.git", branch: "master"
-  gem "fix-db-schema-conflicts", github: "thepracticaldev/fix-db-schema-conflicts", branch: "master"
+  gem "faker", github: "stympy/faker", ref: "e757835"
```

Comment:
since you submit a separate PR for faker and fix-db-schema-conflicts this should probably be removed from this PR

Reply:
`foreman` and `faker` use incompatible versions of `thor` so this work has to be based off of #2142.

Reply:
Never mind. I was wrong and have removed the faker update from this PR.

Reply:
I just remembered that foreman shouldn't be part of the Gemfile: 

> Ruby users should take care not to install foreman in their project's Gemfile.

from https://github.com/ddollar/foreman#installation

Reply:
Interesting. I was just looking back through the history of that warning and it seems to step from the problem of [dependency version conflicts](https://github.com/ddollar/foreman/pull/437#issuecomment-41110303). `foreman` is a development dependency so I think it should in fact be listed in the Gemfile. Dependency version conflicts is already a potential issue with all the other gems anyways.

Reply:
I [proposed](https://github.com/ddollar/foreman/pull/728) an updated phrasing.